src/index.ts

```diff
@@ -46,9 +46,19 @@ v1.post('/session/create', async (req, res) => {
 
 	const hashedServerId = await bcrypt.hash(serverId, 10);
 
-	const token = jwt.sign({ hashedServerId }, JWT_SECRET, {
-		expiresIn: EXPIRE_TIME_SECS,
-	});
+	const session = new Session();
```

Comment:
does the start field get set automatically? and can we maybe require the hashed_uuid in the constructor so we can ensure its set?

Reply:
It should but sure

Reply:
If the database throws some exception if the hashed_uuid member isnt set then that works too, just want to make sure it doesnt cause any bugs